Mathlib/Analysis/NormedSpace/Multilinear/Basic.lean

```diff
@@ -469,10 +469,40 @@ section
 
 variable {𝕜' : Type*} [NormedField 𝕜'] [NormedSpace 𝕜' G] [SMulCommClass 𝕜 𝕜' G]
 
-theorem opNorm_smul_le (c : 𝕜') (f : ContinuousMultilinearMap 𝕜 E G) : ‖c • f‖ ≤ ‖c‖ * ‖f‖ :=
-  (c • f).opNorm_le_bound (mul_nonneg (norm_nonneg _) (opNorm_nonneg _)) fun m ↦ by
-    rw [smul_apply, norm_smul, mul_assoc]
-    exact mul_le_mul_of_nonneg_left (le_opNorm _ _) (norm_nonneg _)
+@[simp]
+theorem opNorm_smul_eq (f : ContinuousMultilinearMap 𝕜 E G) (c : 𝕜') : ‖c • f‖ = ‖c‖ * ‖f‖ := by
```

Comment:
I don't think this change is useful: the lemma `opNorm_smul_le` is only used to get the normed space structure, and once you have it the general lemma `norm_smul` applies.